webapp/components/User/User.vue

```diff
@@ -1,32 +1,37 @@
 <template>
-  <div class="user" v-if="displayAnonymous">
-    <hc-avatar v-if="showAvatar" class="avatar" />
-    <div>
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:28:12Z" title="Wednesday, January 8th 2020, 4:28:12 pm +01:00">Jan 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,6 +1,6 @@
 <template>
   <div class="user" v-if="displayAnonymous">
-    <hc-avatar v-if="showAvatar" class="avatar" />
+    <user-avatar v-if="showAvatar" class="avatar" />
```


Here, the `avatar` class is used to position the `user-avatar` component. That is indeed the responsibility of the wrapper component (in this case `User`) but following the [rscss guidelines](https://rscss.io/index.html) we would not add a separate class here and instead use the `user-avatar` class directly:
```
.user {
  // other styles
  > .user-avatar {
    // positioning of the avatar
  }
}
```

Reply:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-01-08T15:40:14Z" title="Wednesday, January 8th 2020, 4:40:14 pm +01:00">Jan 8, 2020</time>_

---


And while you're at it... I would very much like to replace the current way of positioning the `avatar` with a more modern solution. :wink:

Instead of:
```
.avatar {
  float: left;
  margin-right: 4px;
  height: 100%;
  vertical-align: middle;
}
```
(which isn't even valid CSS because `height` only works for `block` elements and `vertical-align` only for `inline` – so they rule each other out 😅) we could use `flexbox` for positioning. This is one of these times where I would go ahead and try different things until I'm happy with the solution – if you want I'd be happy to have a look at it together. 🙂